### PR TITLE
add default header

### PR DESCRIPTION
**Pull request by TMysiv**
create header

### CODE REVIEW

**Review by lerik1408 — APPROVED** [review]

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
```
Comment (BillyAleks) [inline src/components/Header.tsx:1]: you don't need to import React after v17

File: src/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+const Header: FC = () => {
+
+  return (
+    <>
+      <AppBar position='static' color='primary'>
+        <Toolbar>
+          <Typography
+            variant='h5'
+            sx={{ flexGrow: 1 }}>
```
Comment (BillyAleks) [inline src/components/Header.tsx:13]: Better to put closing brace on a next line

File: src/components/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+            <IconButton
+              sx={{ padding:0}}
+              color='inherit'>
+              <PermIdentityIcon />
+            </IconButton>
+
+            <Box className={'avatar'}>
+              <img
+                src='https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR4OPBibmFcA0jgIPbN-6wPiy37vbmfQN7SAg&usqp=CAU'
```
Comment (BillyAleks) [inline src/components/Header.tsx:27]: Better to upload an default image, like anonymous avatar.
https://www.freepik.com/free-photos-vectors/default-avatar

File: src/components/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
+import { AppBar, Box, IconButton, Toolbar, Typography } from '@mui/material';
+import PermIdentityIcon from '@mui/icons-material/PermIdentity';
+
+const Header: FC = () => {
```
Comment (BillyAleks) [inline src/components/Header.tsx:5]: if you do not have any logic, inside a function, only render - return directly JSX

File: src/components/Header.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+  return (
+    <>
```
Comment (BillyAleks) [inline src/components/Header.tsx:8]: Is there any reason to use fragment here?

**Review by BillyAleks — CHANGES_REQUESTED** [review]
Thank you. Please check out comments

File: src/components/Header.tsx (hunk 1 above)
Comment (TMysiv) [reply src/components/Header.tsx]: fixed

File: src/components/Header.tsx (hunk 2 above)
Comment (TMysiv) [reply src/components/Header.tsx]: fixed

File: src/components/Header.tsx (hunk 3 above)
Comment (TMysiv) [reply src/components/Header.tsx]: done

File: src/components/Header.tsx (hunk 4 above)
Comment (TMysiv) [reply src/components/Header.tsx]: fixed

File: src/components/Header.tsx (hunk 5 above)
Comment (TMysiv) [reply src/components/Header.tsx]: fixed

**Review by RamboUA — APPROVED** [review]

**Pull request merged**